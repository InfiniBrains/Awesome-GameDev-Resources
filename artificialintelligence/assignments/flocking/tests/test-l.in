1.000 0.000 0.000 0.000 1.000 0.000 0.000 3
0.000 0.500 0.000 0.000
0.000 0.000 1.100 0.000
0.000 -0.500 0.000 0.000
0.125
0.125
