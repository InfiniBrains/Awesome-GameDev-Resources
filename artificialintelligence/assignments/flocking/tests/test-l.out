0.000 0.488 0.000 -0.094
0.138 0.000 1.100 0.000
0.000 -0.488 0.000 0.094
0.001 0.465 0.009 -0.185
0.273 0.000 1.083 0.000
0.001 -0.465 0.009 0.185
